### PR TITLE
Semantic linefeeds added to document

### LINKED ISSUE #42: Adopt semantic line breaks in source code
Many of us in the CSSWG organize our spec source code using semantic line breaks, which is breaking on phrase boundaries, rather than at a particular character limit or not breaking at all. I think this would help the diffs against this document be a lot more understandable and easier to read.

See [Semantic Linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) for an explanation and illustrations of this concept, and [css-text-3 source code](https://github.com/w3c/csswg-drafts/blob/main/css-text-3/Overview.bs#L231) for an example.

### PR DESCRIPTION
**Pull request by cwilso**
This is a content-changeless commit - it only changes linefeeds that disappear in Markdown, so the Markdown file appears exactly the same, but should make future review easier.  (Confirmed with visual comparison and "git diff --word-diff --merge-base fd04e64".)

Fixes #42.

### CODE REVIEW

**Comment by cwilso** [comment]
(Also, I'm on to you - this is just an attempt to break my habit of double-spaces after periods, which mess up semantic linefeeds.)

**Review by tantek — APPROVED** [review]
LGTM

File: Vision/README.md [hunk 1]
```diff
@@ -18,36 +52,81 @@ We will improve the fundamental integrity of the Web platform. The Web will not
...
+our mission is to recognize and embody fundamental values and principles 
+into the architecture of the web. 
+We must become more principled in our execution of the vision of the Web.
 
 # Principles and Values
 
 We must
-* Ensure the Web is trustworthy, by ensuring security and privacy for users. 
+* Ensure the Web is trustworthy, 
+by ensuring security and privacy for users.
```
Comment (fantasai) [inline Vision/README.md:92]: You need to indent this line (and all others that are continuations of a list item). Pick your indentation scheme; Tab and I of course prefer tabs ;) but you might want to go with the more conventional spaces indent.

**Review by fantasai — CHANGES_REQUESTED** [review]
Pretty good for a first shot! A couple of minor comments from my experience.

**Comment by fantasai** [comment]
Also don't forget to Squash merge when you're done!

**Review by tantek — APPROVED** [review]
Good catches @fantasai. Agreed with your suggested changes, especially the breaking of lists into one line per comma-delimited item. Reads much better for editing.

File: Vision/README.md [hunk 2]
```diff
@@ -18,36 +52,81 @@ We will improve the fundamental integrity of the Web platform. The Web will not
...
 # Mission of the W3C
-The fundamental purpose of the W3C is to provide an open forum, where diverse voices from around the world and industries come together, incubate, and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
-
-As the W3C leads the Web forward, our mission is to recognize and embody fundamental values and principles into the architecture of the web.  We must become more principled in our execution of the vision of the Web.
+The fundamental purpose of the W3C is to provide an open forum, 
+where diverse voices from around the world and industries come together, 
+incubate, and build consensus for global standards for Web technologies, 
+including organizations of all sizes 
+(from single-person companies to multi-nationals) 
+and considering public feedback.
```
[the remark this answers is not in the document]
Comment (cwilso) [reply Vision/README.md:88]: I can't say that I disagree with you, but that's for a different day, under the guise of #23.

**Pull request merged**